cgmanifest.json

```diff
@@ -3399,9 +3399,9 @@
         "maven": {
           "artifactId": "material",
           "groupId": "com.google.android.material",
-          "version": "1.10.0",
+          "version": "1.11.0",
           "nuGetId": "Xamarin.Google.Android.Material",
-          "nuGetVersion": "1.10.0.4"
+          "nuGetVersion": "1.11.0"
```

Comment:
there is now a version 1.12.0:

https://github.com/material-components/material-components-android/releases/tag/1.12.0

Reply:
> there is now a version 1.12.0:
> 
> https://github.com/material-components/material-components-android/releases/tag/1.12.0



> there is now a version 1.12.0:
> 
> https://github.com/material-components/material-components-android/releases/tag/1.12.0

Thanks. I know. But I will release 1.11 1st and then few days later 1.12. Skipping is not good it can lead to problems later.